src/test/tests.ts

```diff
@@ -133,22 +127,22 @@ const subscriptionManager = new SubscriptionManager({
 
 // indirect call to support spying
 const handlers = {
-  onSubscribe: (msg: SubscribeMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
-    return Promise.resolve(Object.assign({}, params, {context: msg['context']}));
+  onSubscribe: (msg: OperationMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
```

Comment:
what do u mean here @DxCx ? I only change the type on the onSubscribe tests setup because i Changed it in the server